### PR TITLE
update mobile bundles

### PR DESCRIPTION
**Pull request by MDevster**

### CODE REVIEW

File: linter/allowlists/en.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ assets
 Assistant
 Avatar
 automator
+backend
```
Comment (Toflar) [inline linter/allowlists/en.txt:14]: That's incorrect. The correct English term is `back end` ;)

**Review by Toflar — CHANGES_REQUESTED** [review]
Can you just fix that one issue I mentioned. I'll merge asap then :)

File: linter/allowlists/en.txt (hunk 1 above)
Comment (MDevster) [reply linter/allowlists/en.txt:14]: I adapted it in my description, I did not remove it from the whitelist, because other extensions use it as a keyword.

Comment (Toflar) [reply linter/allowlists/en.txt:14]: Which would be incorrect imho but whatever.

**Review by Toflar — APPROVED** [review]

**Pull request merged**